### DISCUSSION TITLE
Usage only on Pages

### DISCUSSION THREAD

**Discussion by csikoszoltan** [saade/filament-fullcalendar#68] [Q&A] [answered]
Hello,

I would like to use the Calendar widget only on [Pages](https://filamentphp.com/docs/2.x/admin/pages/getting-started).
I succesfully added it, but I have no idea how to remove it from the dashboard widgets.
Is it possible somehow?

Thanks,
Zoli

**Comment by saade** [comment, marked as answer]
Hello,

https://discord.com/channels/883083792112300104/935819320699805737/1012439009944932363

**Reply by csikoszoltan** [reply]
Thank you!
